app/metrics.yaml

```diff
@@ -157,7 +157,7 @@ events:
       - interaction
     notification_emails:
       - android-probes@mozilla.com
-    expires: "2022-12-01"
+    expires: 108
```

Comment:
Glean docs say "X" (where X is the major version...), and use expires: "88" as an example. Is it fine to skip the quotes?

Reply:
Yes, we don't need to put the quotes around the version number.  I got the example from the Glean team here: https://github.com/mozilla/glean/blob/main/samples/android/app/metrics.yaml

Reply:
ok, perfect!

Reply:
Woops, guess we didn't update the docs then